encoders/image/BigTransferEncoder/README.md

```diff
@@ -1,14 +1,24 @@
 # BigTransferEncoder
 
-`BigTransferEncoder` is Big Transfer (BiT) presented by Google (https://github.com/google-research/big_transfer), is a State-of-the-art transfer learning model for computer vision. This class uses pretrained BiT to encode data from an `ndarray`, potentially B x (Channel x Height x Width) into an `ndarray` of shape `B x D`. Internally, :class:`BigTransferEncoder` wraps the models from https://storage.googleapis.com/bit_models/.
+`BigTransferEncoder` is [Big Transfer (BiT) presented by Google](https://github.com/google-research/big_transfer), is a State-of-the-art transfer learning model for computer vision. This class uses pretrained BiT to encode data from an `ndarray`, potentially B x (Channel x Height x Width) into an `ndarray` of shape `B x D`. Internally, :class:`BigTransferEncoder` wraps the models from [here](https://storage.googleapis.com/bit_models/).
```

Comment:
**Old**
`BigTransferEncoder` is [Big Transfer (BiT) presented by Google](https://github.com/google-research/big_transfer), is a State-of-the-art transfer learning model for computer vision.

**New**
`BigTransferEncoder` from [Big Transfer (BiT)](https://github.com/google-research/big_transfer) is Google's state-of-the-art transfer learning model for computer vision.